oshmem/mca/atomic/ucx/atomic_ucx_cswap.c

```diff
@@ -19,14 +19,14 @@
 
 #include "atomic_ucx.h"
 
-/* nlong argument should be constant to hint compiler
- * to calculate nlong relative branches in compile time */
+/* size argument should be constant to hint compiler
+ * to calculate size relative branches in compile time */
 static inline
 int mca_atomic_ucx_cswap_inner(void *target,
                                void *prev,
                                const void *cond,
                                const void *value,
-                               size_t nlong,
+                               size_t size,
```

Comment:
can we pass "cond" and "value" as uint64_t as optimization?
same for other functions..